example/spec/parallel/cnmWorkflow/data/rules/kinesis/unique/L2_HR_PIXC_kinesisRule.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
This is just moved to a new directory

Reply:
Yeah, I did it this way because we're setting rule overrides. We can do that though so long as the json files are in separate directories, even if they're nested. I re-arranged to leave the original in place and create the new one in a new subdirectory. It works because `addRules` doesn't scan for files in subdirectories.

https://github.com/nasa/cumulus/pull/4037/commits/a5db113cc0c66f991acd33ec04e6470bc8507c69

There's an alternative here where we can update `addRules` to support multiple overrides for multiple files but that seems like overkill.